source/install-guide/installing-from-the-installer.rst

```diff
@@ -0,0 +1,162 @@
+.. _installing-from-the-installer:
```

Comment:
The community is not in capacity of building the installer (see https://github.com/geonetwork/core-geonetwork/pull/4961) , so it makes more sense to move that doc to the place where people get this installer https://my.geocat.net ?

Reply:
Yes, I just discussed that with @josegar74 and I will remove this section.